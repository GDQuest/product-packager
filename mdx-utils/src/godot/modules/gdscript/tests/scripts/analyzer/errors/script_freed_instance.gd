class A extends Node:
	pass

func test():
	var x = A.new()

	x.free()

	var ok = x
	var bad: A = x
