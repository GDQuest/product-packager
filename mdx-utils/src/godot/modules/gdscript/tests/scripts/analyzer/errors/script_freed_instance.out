GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> analyzer/errors/script_freed_instance.gd
>> 10
>> Trying to assign invalid previously freed instance.
